frame/support/procedural/src/lib.rs

```diff
@@ -1438,3 +1441,28 @@ pub fn origin(_: TokenStream, _: TokenStream) -> TokenStream {
 pub fn composite_enum(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
+
+#[proc_macro_attribute]
+pub fn export_section(attr: TokenStream, tokens: TokenStream) -> TokenStream {
+	match macro_magic::mm_core::export_tokens_internal(attr, tokens, false) {
+		Ok(tokens) => tokens.into(),
+		Err(err) => err.to_compile_error().into(),
+	}
+}
+
+#[import_tokens_attr]
+#[proc_macro_attribute]
+pub fn import_section(attr: TokenStream, tokens: TokenStream) -> TokenStream {
+	let foreign_mod = parse_macro_input!(attr as ItemMod);
+	let mut internal_mod = parse_macro_input!(tokens as ItemMod);
+
+	if let Some(ref mut content) = internal_mod.content {
+		if let Some(foreign_content) = foreign_mod.content {
+			content.1.extend(foreign_content.1);
+		}
+	}
```

Comment:
You'll want to have errors for the case where the mod this is attached to _isn't_ a pallet, and the case where the mod this is attached to has no body/contents (though requiring that this can only be attached to a pallet might resolve the second issue as well so just the first one should be necessary)

Reply:
@sam0x17 Added a check for the number of attributes. What would be the best way to check for pallet macro?